### ISSUE TITLE
Hostd 0.1.6

### ISSUE THREAD

**Issue by azrielitay** [SiaFoundation/hostd#159]
### Current Behavior

 Cannot login to web GUI ,Get error password wrong. 
 

### Expected Behavior

Login to  web GUI

### Steps to Reproduce

upgrade to hostd 0.1.6 enter any password with digits.
enter the password in the web GUI

### Version

0.1.6

### What operating system did the problem occur on (e.g. Ubuntu 22.04, macOS 12.0, Windows 11)?

 Windows 11

### Anything else?

CLI Verbus doesn't show time correctly. in the beginning of the line.

**Comment by NickJH** [comment]
WFM on Linux (Ubuntu)

**Comment by n8maninger** [comment]
Fixed by 8c665ca. We'll put out a new release immediately

**Comment by NickJH** [comment]
FWIW, i can start hostd and logon throuth the UI but I get:
```
root@hostd-test:~/v0.1.6# systemctl restart hostd
root@hostd-test:~/v0.1.6# hostd status
Please choose a password to unlock the UI and API.
(The password must be at least 4 characters.)
Enter password:

Type in your 12-word seed phrase and press enter. If you do not have a seed phrase yet, type 'seed' to generate one.
```

**Comment by NickJH** [comment]
NVM. It should be `systemctl status hostd` only. I was falling back to my `siad` riits.

**Comment by n8maninger** [comment]
@azrielitay we just released v0.1.7 with this fixed. Thanks for reporting!
